webapp/channels/src/packages/mattermost-redux/src/store/configureStore.ts

```diff
@@ -46,7 +46,13 @@ export default function configureStore<S extends GlobalState>({
         autoPause: true,
     });
 
-    const middleware = applyMiddleware(thunk);
+    const middleware = applyMiddleware(
+
+        // @hmhealey I've added this extra argument to Thunks to store information related to the store that can't be
```

Comment:
May be we should add this file in CODEOWNERS as well

Reply:
Yeah, I can do that. It's probably good for us to know if anyone's adding any middleware or anything else we should know about